@xen-orchestra/log/logger.test.js

```diff
@@ -0,0 +1,154 @@
+'use strict'
+
+const { beforeEach, describe, it } = require('node:test')
+const { configure } = require('@xen-orchestra/log/configure')
+const { createCaptureTransport } = require('@xen-orchestra/log/capture')
+const { createLogger } = require('@xen-orchestra/log')
+const assert = require('node:assert/strict')
+
+describe('logger', () => {
+  const logger = createLogger('test-logger')
+
+  const logsTransportDefault = []
+  const transportTest = message => {
```

Comment:
Be consistent with the other transport.

```suggestion
  const logsTransportDebug = []
  const transportDebug = message => {
```